### PR TITLE
docs: clarify project support policy in SUPPORT.md

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

Microsoft suggested a standard support policy format for open source projects. It's not set in stone yet, but this implements the current suggestion.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the [existing issues][github-issues] before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new Issue.
+
+For help and questions about using this project, please open a new GitHub issue or visit the [#aks-engine-users][aks-engine-users-slack] Kubernetes Slack channel. The AKS Engine project maintainers will respond to the best of their abilities.
+
+## Microsoft Support Policy
+
+Support for this project is limited to the resources listed above.
+
+[aks-engine-users-slack]: https://kubernetes.slack.com/archives/CU3N85WJK
```
Comment (marosset) [inline SUPPORT.md:13]: Can you move these 2 points under to under the 'For help and questions' section?

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the [existing issues][github-issues] before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new Issue.
+
+For help and questions about using this project, please open a new GitHub issue or visit the [#aks-engine-users][aks-engine-users-slack] Kubernetes Slack channel. The AKS Engine project maintainers will respond to the best of their abilities.
```
Comment (marosset) [inline SUPPORT.md:7]: The please open a new GitHub issue seems redundant.
Maybe update this section to:

For help and questions about using the project please
- Checkout out our [FAQ](https://github.com/Azure/aks-engine/blob/master/docs/faq.md)
- Review our [Documentation](https://github.com/Azure/aks-engine/blob/master/docs/README.md)
- Reach out on [aks-engine-users-slack](https://kubernetes.slack.com/archives/CU3N85WJK)

File: SUPPORT.md (hunk 1 above)
Comment (mboersma) [reply SUPPORT.md:15]: 👍 Done.

File: SUPPORT.md (hunk 2 above)
Comment (mboersma) [reply SUPPORT.md]: 👍 Agreed, much better.

**Review by marosset — APPROVED** [review]

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Comment by mboersma** [comment]
Let's wait to merge this until @craiglpeters has a chance to review it.

**Review by craiglpeters — APPROVED** [review]
LGTM

**Pull request merged**